packages/utils/src/lib/reports/generate-stdout-summary.ts

```diff
@@ -73,8 +73,10 @@ function reportToOverviewSection({
   plugins,
 }: ScoredReport): string {
   const table = new CliTable3({
+    // eslint-disable-next-line no-magic-numbers
+    colWidths: [67, 7, 8],
     head: reportRawOverviewTableHeaders,
-    colAligns: ['left', 'right', 'right'],
+    colAligns: ['left', 'right', 'left'],
```

Comment:
Numbers should be aligned right.

```suggestion
    colAligns: ['left', 'right', 'right'],
```